components/tyk-dashboard/values.yaml

```diff
@@ -0,0 +1,166 @@
+global:
+  license:
+    dashboard: ""
+
+  servicePorts:
+    dashboard: 3000
+    gateway: 8080
```

Comment:
can you also write a brief description about these lines? i know they are exploratory but having a comment might be useful